src/components/layout/PageLayout/index.tsx

```diff
@@ -9,6 +10,7 @@ interface LayoutProps {
 
 const PageLayout = ({ user, children }: PropsWithChildren<LayoutProps>) => (
   <>
+    <SEO />
```

Comment:
Let's move this component to the `src/pages/_app.tsx` file for all technical components that aren't directly related to the UI layout.